packages/core/bootstrap/src/index.ts

```diff
@@ -79,13 +80,38 @@ const withStatusCode: Middleware = async (execute, context) => async (input) =>
 
 // Log adapter input & output data
 const withLogger: Middleware = async (execute, context) => async (input: AdapterRequest) => {
+  interface IError {
+    jobRunID: string
+    params: AdapterRequestData
+    message: string
+    feedID: string
+    endpoint?: string
+    errorResponse?: any
+    rawError?: any
+    stack: any
+  }
   Logger.debug('Input: ', { input })
   try {
     const result = await execute(input, context)
     Logger.debug(`Output: [${result.statusCode}]: `, { output: result.data })
     return result
   } catch (error) {
-    Logger.error(error.toString(), { stack: error.stack })
+    const feedID = metrics.util.getFeedId(input)
+    const errorLog: IError = {
+      message: error.toString(),
+      jobRunID: input.id,
+      params: input.data,
+      feedID,
+      stack: error.stack,
```

Comment:
Similar to log level trace below, stack is very verbose. 
We should only include it on `.level === 'debug'`.

And the `rawError` also includes stack trace right? So don't include `stack` when level is trace.

Reply:
resolved in #1058